### ISSUE TITLE
New contstraint

### ISSUE THREAD

**Issue by b-connect** [talkdirty/android_job_scheduler#7]
Is is possible to create a new constraint, for running jobs only in background?

**Comment by talkdirty** [comment]
Yes, this could be implemented quite trivially, and also seems like a
useful addition. I'll add it sometime later this week if I get to it
